index.js

```diff
@@ -119,6 +120,12 @@ const createProject = async function({ contract, frontend, projectDir, veryVerbo
     await replaceInFiles({ files: `${projectDir}/README.md`, from: /npm\b( run)?/g, to: 'yarn' })
   }
 
+  // setup rust
+  let wasRustupInstalled = false;
+  if (contract == 'rust') {
```

Comment:
Please use triple equals everywhere. Double equals [does weird things](https://blog.webdevsimplified.com/2020-08/==-vs-===/).

```suggestion
  if (contract === 'rust') {
```